### PR TITLE
Remove period at the end of lesson keys in csf

### PR DESCRIPTION
**Pull request by dmcavoy**
Names of lessons were showing that they could not be translated for some lessons. After investigating we determined it was because of a period in the key for the lesson. This removes periods from names of existing lessons that were causing issues to unblock curriculum writers.

Thanks @bethanyaconnor for your help investigating this.

## Links

[Slack Convo](https://codedotorg.slack.com/archives/C02EEGWLHR8/p1644350606411229)

## Testing story

- Ran `bundle exec rake build` to make sure things build ok
- Checked that these lessons no longer show the translation error message

## Follow-up work

- [Make sure when new lessons are made or lessons are copied we do not end up with periods in the key](https://codedotorg.atlassian.net/browse/PLAT-1594)

### CODE REVIEW

File: dashboard/config/scripts_json/csa-self-paced-pl.script_json [hunk 1]
```diff
@@ -36,7 +36,7 @@
   ],
   "lessons": [
     {
-      "key": "Navigating the Code.org learning platform",
+      "key": "Navigating the Codeorg learning platform",
```
Comment (megcrenshaw) [inline dashboard/config/scripts_json/csa-self-paced-pl.script_json:39]: Will this also get messed up? Or is it only at the *end* of the key?

**Review by megcrenshaw — COMMENTED** [review]
When I do a regex search in RubyMine for `"lesson\.key": ".*\."` I get matches from `coursea`, `courseb`, and `coursec`. If the period is also not allowed within the key itself, I'll re-search

**Review by bethanyaconnor — APPROVED** [review]
Thanks for fixing these!

File: dashboard/config/scripts_json/csa-self-paced-pl.script_json (hunk 1 above)
Comment (dmcavoy) [reply dashboard/config/scripts_json/csa-self-paced-pl.script_json:39]: This was also an issue. You can see Hannah linked to it in the slack thread

**Comment by dmcavoy** [comment]
> When I do a regex search in RubyMine for `"lesson\.key": ".*\."` I get matches from `coursea`, `courseb`, and `coursec`. If the period is also not allowed within the key itself, I'll re-search

Its not allowed anywhere in the key it looks like.

**Comment by dmcavoy** [comment]
@megcrenshaw feel free to do another search and report back on findings. I'm going to merge this to unblock writers.

**Comment by megcrenshaw** [comment]
@dmcavoy Thanks for the clarification! I'm also seeing it in `cspassessment.script_json` ("lesson.key": "NOTE: Deprecated. Teacher cannot see results. CSP Culminating Assessment") and `equity-pd.script_json` ("lesson.key": "Code.org Equity PD"). That's all I see. Searching `"lesson\.key": ".*\..*"` in RubyMine

**Comment by dmcavoy** [comment]
@megcrenshaw thank you! Here is the follow up PR https://github.com/code-dot-org/code-dot-org/pull/44742

**Pull request merged**